### PR TITLE
Translations: edits for capitalization and consistency

### PR DESCRIPTION
**Pull request by sabernhardt**
1. Capitalize proper nouns: African, YouTube, and Acropolis of Athens.
2. [Use lowercase t](https://developer.wordpress.org/apis/internationalization/internationalization-guidelines/#descriptions) for `translators:` comments.
3. Add a period in the context **when it matches another string**.
4. Use plural "IRL events" in `services-subscriber-only-section.php` for consistency with the string in pricing patterns.

### CODE REVIEW

File: patterns/footer-centered.php [hunk 1]
```diff
@@ -29,7 +29,7 @@
 	<p class="has-text-align-center has-small-font-size">
 		<?php
 		printf(
-			/* Translators: Designed with WordPress. %s: WordPress link. */
+			/* translators: Designed with WordPress. %s: WordPress link. */
```
Comment (sabernhardt) [inline patterns/footer-centered.php:32]: Capitalized "Translators:" apparently works correctly, too.
https://translate.wordpress.org/projects/wp-themes/twentytwentyfive/fr/default/?filters%5Bterm%5D=Designed+with

File: patterns/services-subscriber-only-section.php [hunk 2]
```diff
@@ -28,7 +28,7 @@
 				<!-- /wp:list-item -->
 
 				<!-- wp:list-item {"fontSize":"medium"} -->
-				<li class="has-medium-font-size"><?php esc_html_e( 'Join our IRL event.', 'twentytwentyfive' ); ?></li>
+				<li class="has-medium-font-size"><?php esc_html_e( 'Join our IRL events.', 'twentytwentyfive' ); ?></li>
```
Comment (sabernhardt) [inline patterns/services-subscriber-only-section.php:31]: Combines similar strings:
https://translate.wordpress.org/projects/wp-themes/twentytwentyfive/fr/default/?filters%5Bterm%5D=IRL

File: patterns/template-home-with-sidebar-news-blog.php [hunk 3]
```diff
@@ -85,7 +85,7 @@
 						<div class="wp-block-group has-small-font-size">
 							<!-- wp:post-terms {"term":"category","style":{"typography":{"textTransform":"uppercase","letterSpacing":"1.4px"}}} /-->
 							<!-- wp:paragraph -->
-							<p><?php echo esc_html_x( '·', 'Separator between date and categories', 'twentytwentyfive' ); ?></p>
```
Comment (sabernhardt) [inline patterns/template-home-with-sidebar-news-blog.php:88]: Without the period at the end of the context, this created a separate string.
https://translate.wordpress.org/projects/wp-themes/twentytwentyfive/fr/default/?filters%5Bterm%5D=Separator+between+date+and+categories

File: patterns/template-query-loop-vertical-header-blog.php [hunk 4]
```diff
@@ -44,7 +44,7 @@
 
 	<!-- wp:query-no-results -->
 		<!-- wp:paragraph -->
-		<p><?php echo esc_html_x( 'Sorry, but nothing was found. Please try a search with different keywords.', 'Message explaining that there are no results returned from a search', 'twentytwentyfive' ); ?></p>
```
Comment (sabernhardt) [inline patterns/template-query-loop-vertical-header-blog.php:47]: Matches the context to the more common version, with the period.
https://translate.wordpress.org/projects/wp-themes/twentytwentyfive/fr/default/?filters%5Bterm%5D=Sorry%2C+but+nothing+was+found

**Comment by carolinan** [comment]
If I understand correctly, all text string changes require approval from the polyglots team after RC1.

File: patterns/footer-centered.php (hunk 1 above)
Comment (carolinan) [reply patterns/footer-centered.php:32]: Interesting. Maybe there are other tools where it does not work. I don't know the reason behind this requirement.

**Comment by sabernhardt** [comment]
I'll mention this in Slack #polyglots.

[My spreadsheet](https://docs.google.com/spreadsheets/d/1JQgQNC5rbmDivcMUTZl4WfPonJUpAPGVOle6_LbZICQ/edit?usp=sharing) might not have much value now, but I should share it somewhere.
- The first sheet groups the strings from `functions.php` and from within the content of patterns.
- The second sheet contains all 489 strings from translate.wordpress.org.

**Review by carolinan — APPROVED** [review]
I have applied the PR and confirmed that there were no block validation errors in the updated patterns.

**Review by audrasjb — APPROVED** [review]
As a GTE and Locale Manager for fr_FR (and core committer) I approve these changes. Some of them are not super important (like `Translators` -> `translators`) but some other changes are more relevant for a post-String freeze inclusion.

**Comment by juanfra** [comment]
Thank you all!

**Pull request merged**